docs/Modules/ecdsa/ecdsasdk.mdx

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_label: "SDK Usage"
+sidebar_position: 3
+description: "Learn to integrate the ECDSA Module using Biconomy SDK"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# ECDSA Ownership Module Integration Guide
+
+## Introduction 📘
+
+Integrate the ECDSA Validation Module into Biconomy Smart Accounts using the SDK.
+
+## Setup and Installation 🛠️
+
+<Tabs>
+<TabItem value="js" label="npm">
+
+```bash
+
+npm install @biconomy/modules ethers
+
+```
+
+</TabItem>
+<TabItem value="py" label="yarn">
+
+```bash
+
+yarn add @biconomy/modules ethers
+
+```
+
+</TabItem>
+<TabItem value="java" label="pnpm">
+
+```bash
+
+pnpm add @biconomy/modules ethers
+
+```
+
+</TabItem>
+</Tabs>
+
+## Creating a Signer 🗝️
+
+```javascript
+import { ethers } from "ethers";
+
+const provider = new ethers.providers.JsonRpcProvider("[RPC_Endpoint]");
+const signer = new ethers.Wallet("[Private_Key]", provider);
+```
+
+## Importing and Initializing ECDSA Module 🌟
+
+```javascript
+import {
+  ECDSAOwnershipValidationModule,
+  DEFAULT_ECDSA_OWNERSHIP_MODULE,
+} from "@biconomy/modules";
+
+const ecdsaModule = await ECDSAOwnershipValidationModule.create({
+  signer: signer,
+  moduleAddress: DEFAULT_ECDSA_OWNERSHIP_MODULE,
+});
+```
+
+## Creating a Smart Account 🚀
+
+```javascript
+import { BiconomySmartAccountV2 } from "@biconomy/account";
+
+const smartAccount = await BiconomySmartAccountV2.create({
+  signer: signer,
+  defaultValidationModule: ecdsaModule,
+  // Additional configurations
+});
+```
+
+:::info
+The Smart Account's address is counterfactual, its address can be calculated before deployment. This generated address depends on the default validation module specified during account creation, here ECDSA Module.
```

Comment:
SA's address depends on initialization data for the module, in this case, on the signer's public key (ower address).

Reply:
Hope it's fine now